session_rpcserver.go

```diff
@@ -11,6 +12,21 @@ import (
 	"github.com/lightninglabs/lightning-node-connect/mailbox"
 	"github.com/lightninglabs/lightning-terminal/litrpc"
 	"github.com/lightninglabs/lightning-terminal/session"
+	"github.com/lightninglabs/lndclient"
+	"github.com/lightningnetwork/lnd/keychain"
+)
+
+var (
+	// TODO(elle): Temp values. change these before landing pr.
+	nBytes, _ = hex.DecodeString(
```

Comment:
Should instead be a key derived from the user's seed?

Reply:
from the users seed in LND? yes this is the idea. So we have a static pubkey here (need to still gen a proper one) and then we derive a shared secret with lnd using this static key and a seed in LND using the `DeriveSharedSecret` endpoint. the macaroon service does this. 
Is this what you mean or is there a litd user seed that you mean?

Reply:
Here's the public key derived with `numsgen` from LNC with the phrase "Shared Secret": 0215b5a3e0ef58b101431e1e513dd017d1018b420bd2e89dcd71f45c031f00469e
Should probably mention the phrase and link to https://github.com/lightninglabs/lightning-node-connect/tree/master/mailbox/numsgen